tests/system/test_sending.py

```diff
@@ -48,7 +48,7 @@ def wait_for_trash(client, thread_id):
 @pytest.mark.parametrize("client", all_accounts)
 def test_sending(client):
     # Create a message and send it to ourselves
-    subject = "%s (Self Send Test)" % strftime("%Y-%m-%d %H:%M:%S")
+    subject = "{} (Self Send Test)".format(strftime("%Y-%m-%d %H:%M:%S"))
```

Comment:
Note that this not make much sense because strftime needs more arguments but those tests are not even run... They are broken and they were ignored in pytest.ini before we even started maintaining this project. They will most probably be chopped at some point, but on a separate PR/discussion.